packages/lesswrong/lib/collections/tags/fragments.ts

```diff
@@ -18,6 +18,9 @@ registerFragment(`
     wikiGrade
     createdAt
     wikiOnly
+    tagFlagsIds
```

Comment:
Does this need to be in `TagBasicInfo`? This potentially adds a decent amount of data to the front page download.

Reply:
It'll be an array of up to, max, let's 20 id's, but I'd expect usually under 5. 

Also doesn't seem necessary to have on the frontpage.

Reply:
It's a bit hard. If you don't have it here, a bunch of filtering on other parts of the side become a bit janky. I could rearchitect a bunch of Fragments, but it seemed overall riskier than just adding it here, given that it's going to just be an array of 2-3 ids at worst, and most of the time just an empty array.

Reply:
This field has literally zero references in the components folder.

Reply:
It may not seem like much, but you have to multiply by the number of tags applied to posts that are visible on the front page. I also assign non-negligible probability to this hitting an Apollo performance weak spot. If this was actually needed, I'd say we should do the benchmarking to confirm that it's actually negligible. But it appears to be unused anyways.

Reply:
Huh, yeah. Looks like I removed all the usages. Let's remove it then.